it/v1/endpoints/ListTransactionsControllerISpec.scala

```diff
@@ -68,65 +69,6 @@ class RetrieveTransactionsControllerISpec extends IntegrationBaseSpec {
     """.stripMargin
   )
 
-  private val mtdJson = Json.parse(
```

Comment:
I don't think it's necessary to pull this JSON into the fixtures when it is only being used in a single place.